ethereum/datastructures/src/main/java/tech/pegasys/teku/datastructures/util/CommitteeUtil.java

```diff
@@ -50,10 +50,15 @@
    * @param index
    * @param index_count
    * @param seed
+   * @deprecated CommitteeUtil should be accessed via Spec.getCommitteeUtil().computeShuffledIndex()
    * @return
    * @see
    *     <a>https://github.com/ethereum/eth2.0-specs/blob/v0.8.0/specs/core/0_beacon-chain.md#is_valid_merkle_branch</a>
    */
+  /**
+   * @deprecated CommitteeUtil should be accessed via Spec.getCommitteeUtil().getAggregatorModulo()
```

Comment:
nit: wrong method referenced
```suggestion
   * @deprecated CommitteeUtil should be accessed via Spec.getCommitteeUtil().computeShuffledIndex()
```